docs/user/openstack/README.md

```diff
@@ -33,7 +37,51 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ```
 
-* Swift must be enabled.  The user must have `swiftoperator` permissions and
+If you choose to use environment variables in place of a clouds.yaml, or along side it, consult the following doccumentation:
+https://www.terraform.io/docs/providers/openstack/#configuration-reference
+
+
+## OpenStack Requirements
+
+### Recommended Minimums
+
+With the latest version of the installer, we recommend the following minimums in order to get a succesful deployment.
+
+   * OpenStack Quota
+     * Floating IPs
+       * Minimum: 2
```

Comment:
I've just realised, the OpenStack router uses up one floating IP as well. So we need 1 for the router, another for the bootstrap and third for the API. So: minimum 3 recommended 4+? I'd actually recommend one for each node so let's make it an even 10?

Reply:
wait what?

Reply:
The one for the OpenStack router is added at the end of deployment when the bootstrap FIP is supposed to be deleted already.

It's correct that we need 3 FIPs, but only 2 at the same time. Minimal requirement should say 2 FIPs.